### PR TITLE
New cloaks KB page, squashed into one commit.

### PR DESCRIPTION
**Pull request by bigpresh**
All history and commit messages illustrating its route to this point lost, by
request.

### CODE REVIEW

**Comment by bigpresh** [comment]
Nudge?

**Comment by AbstractBeliefs** [comment]
Can I be cheeky and suggest at this late stage you include a sentence or two about bot cloaks? What they mean, how to get them, and what they look like?

File: content/kb/general/cloaks.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  you could reveal your IP to them
+
+For these reasons, we advise you to consider cloaks as only very basic
+protection from casual observers, and a way to stop your IP/hostname being
+passively logged in most cases, but caution that they cannot be relied upon to
+hide your IP/hostname robustly - if you want that, you should consider an
+[IRC bouncer](https://en.wikipedia.org/wiki/IRC_bouncer),
+[VPN](https://en.wikipedia.org/wiki/Virtual_private_network)
+[Tor](https://www.torproject.org/) (see our blog post on 
+[connecting to freenode via Tor](https://freenode.net/news/tor-online).
```
Comment (AbstractBeliefs) [inline content/kb/general/cloaks.md]: Do we have mismatched closing ) here? I see one opening on line 70 "(see our blog post", followed by 1 opening and 1 closing on 71 for the blog link.

File: content/kb/general/cloaks.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Cloaks do not effectively hide your IP
+======================================
+Cloaks can help obscure your IP address/hostname from casual observers, but
+should *not* be relied upon for that purpose, as they are not reliable:
+
+- Connecting before identifying to NickServ (or whilst services are unavailable
+  due to a netsplit or maintenance) will show your uncloaked IP/hostname
+  connecting via SASL can help a little here, as authentication is done much
+  earlier in the connection)
```
Comment (AbstractBeliefs) [inline content/kb/general/cloaks.md]: Extra closing ) here?

**Comment by edk0** [comment]
closing as merged

**Pull request closed without merging**